src/backend/distributed/citus.control

```diff
@@ -1,6 +1,6 @@
 # Citus extension
 comment = 'Citus distributed database'
-default_version = '11.0-1'
+default_version = '11.0-2'
```

Comment:
I guess we wouldn't need most of the 11.0-2 related changes if we include the fix on 11.0-1 right?

Reply:
yes